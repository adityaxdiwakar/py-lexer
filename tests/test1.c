int main(){
  int a;
  a+=3;
  scanf("%d%d",a);
  return 0;
}
