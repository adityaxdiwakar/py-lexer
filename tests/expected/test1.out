INT_WORD
IDENT(main)
LPAREN
RPAREN
LBRACK
INT_WORD
IDENT(a)
SEMI
IDENT(a)
PLUS
EQUALS
INT(3)
SEMI
IDENT(scanf)
LPAREN
STR(%d%d)
COMMA
IDENT(a)
RPAREN
SEMI
RETURN
INT(0)
SEMI
RBRACK
